addons/surgery/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -112,4 +118,74 @@ class ACE_Medical_Treatment_Actions {
         callbackProgress = QFUNC(npwtTreatmentProgress);
         callbackSuccess = "";
     };
+    class Ultrasound: BasicBandage {
+        displayName = CSTRING(Ultra_Use);
+        displayNameProgress = CSTRING(Ultra_Action);
+        category = "surgery";
+        treatmentLocations = QGVAR(surgicalLocation);
+        allowedSelections[] = {"Body"};
+        allowSelfTreatment = 0;
+        medicRequired = QGVAR(surgicalAction_MedLevel);
+        treatmentTime = QGVAR(intermediateTime);
+        items[] = {"kat_ultrasound"};
+        condition = QUOTE(!(_patient getVariable [ARR_2(QQGVAR(imaging),false)]));
```

Comment:
IMO it should be possible to do imaging everytime
i am just testing this PR, i did this action and i cant do it again until i perform pericardial tap, assesment should be possible anytime